go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -125,27 +125,22 @@ func (oc *Controller) updateEgressFirewall(oldEgressFirewall, newEgressFirewall
 func (oc *Controller) deleteEgressFirewall(egressFirewall *egressfirewallapi.EgressFirewall) []error {
 	klog.Infof("Deleting egress Firewall %s in namespace %s", egressFirewall.Name, egressFirewall.Namespace)
 
-	nsInfo, err := oc.waitForNamespaceLocked(egressFirewall.Namespace)
-	if err != nil {
-		return []error{fmt.Errorf("failed to wait for namespace %s event (%v)",
-			egressFirewall.Namespace, err)}
+	nsInfo := oc.getNamespaceLocked(egressFirewall.Namespace)
+	if nsInfo != nil {
+		// clear it so an error does not prevent future egressFirewalls
+		nsInfo.egressFirewallPolicy = nil
+		nsInfo.Unlock()
 	}
-	defer nsInfo.Unlock()
-
-	// copy the egressFirewall object out of nsInfo and clear it so an error does not prevent future egressFirewalls
-	// from admission
-	egressFirewallCopy := nsInfo.egressFirewallPolicy
```

Comment:
this was not copying because it is a pointer, right?
https://play.golang.org/p/UuC-DQ1uYLt

Reply:
yeah, I don't think we need to do the copy anyway.